app/src/main/java/com/bebetterprogrammer/tictactoe/activities/PlayWithJarvisActivity.kt

```diff
@@ -84,12 +84,12 @@ class PlayWithJarvisActivity : AppCompatActivity() {
             finish()
         })
     }
+
     fun moveFirst(flag: Int, flag1: Int, flag2: Int) {
-        Toast.makeText(this, " " + flag + " " + flag1 + " " + flag2, Toast.LENGTH_SHORT).show()
-//        val intent = Intent(this@PlayWithJarvisActivity, ::class.java).apply {
-//            putExtra("Flag", flag)
-//            putExtra("Flag1", flag)
-//            putExtra("Flag2", flag)
-//            startActivity(intent)
+        val intent = Intent(this, VsJarvis::class.java)
```

Comment:
VsJarvis class is not needed as we can use the existing class GamePlayActivity.kt with Jarvis variant. Just pass the variant via intent and take the decision based on that variant.